docs/pages/router/layouts/settings.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: Settings
+---
+
+:::caution
+
+This guide refers to upcoming Expo Router features which are experimental, not covered by semantic versioning and you may need to be using the latest version on `main`
+
+:::
```

Comment:
```suggestion
> **warning** This guide refers to upcoming Expo Router features which are experimental, not covered by semantic versioning and you may need to be using the latest version on `main`.
```

This is a proprietary Docusaurus admonitions syntax, we do not support it there. Instead, we can use GH syntax inspired themed callouts.